src/test/java/org/apache/drill_web_test_framework/web_ui/tests/secure/login/LoginTest.java

```diff
@@ -14,25 +14,24 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package org.apache.drillui.test.framework.testng.secure.login;
+package org.apache.drill_web_test_framework.web_ui.tests.secure.login;
 
-import org.apache.drillui.test.framework.initial.TestProperties;
-import org.apache.drillui.test.framework.testng.secure.BaseSecureTest;
+import org.apache.drill_web_test_framework.properties.TestProperties;
+import org.apache.drill_web_test_framework.web_ui.steps.AuthSteps;
+import org.apache.drill_web_test_framework.web_ui.tests.secure.BaseSecureTest;
+import org.testng.Assert;
 import org.testng.annotations.Test;
-import org.apache.drillui.test.framework.steps.webui.AuthSteps;
-
-import static org.testng.Assert.*;
 
 public class LoginTest extends BaseSecureTest {
   @Test(groups = {"functional"})
   public void testLogin() {
-    assertEquals(
+    Assert.assertEquals(
```

Comment:
It will be better to use static import in testLogin() and testLogout() methods for the assertEquals() method and remove Assert class name from tests.

Reply:
@denysord88  Already fixed in scope of MD-4805_AdminsView_VS_UsersView_And_REFACTORING #36